File: solutions/beecrowd/1159/1159.clj
(ns main)

(defn sum-next-five-even [number]
  (->> (range number (+ number 10))
       (filter even?)
       (reduce +)))

(defn main []
  (loop [x (Integer/parseInt (read-line))]
    (when (not= x 0)
      (println (sum-next-five-even x))
      (recur (Integer/parseInt (read-line))))))

(main)

File: solutions/beecrowd/1159/1159.py
import sys


def sum_five_consecutive_even_numbers(number):
    return sum(filter(lambda x: x % 2 == 0, range(number, number + 10)))


for line in sys.stdin:
    number = int(line)
    if number:
        print(sum_five_consecutive_even_numbers(number))
